spec/support/shared_examples/webauthn_setup.rb

```diff
@@ -38,4 +40,44 @@
 
     expect_webauthn_setup_error
   end
+
+  context 'platform authenticator logging' do
+    before do
+      allow(IdentityConfig.store).to receive(:platform_auth_set_up_enabled).and_return(true)
```

Comment:
Depending which lands first between this and #9485, the other will need to rebase to account.

Reply:
I can rebase my work. Everything _looks_ simple to reconcile.

Reply:
Yeah, I'd expect we can just remove the reference to `platform_auth_set_up_enabled` here now.

```suggestion
```